documentation/docs/install-pmm/install-pmm-server/baremetal/helm/index.md

```diff
@@ -1,166 +1,173 @@
 
-Please ensure proper locality either by physically separating workloads in Kubernetes clusters or running separate Kubernetes clusters for the databases and monitoring workloads.
+## Deployment best practices
 
-You can physically separate workloads by properly configuring Kubernetes nodes, affinity rules, label selections, etc.
+For optimal monitoring:
+{.power-number}
 
-Also, ensure that the Kubernetes cluster has [high availability](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) so that in case of a node failure, the monitoring service will be running and capturing the required data.
+1. Separate PMM Server from monitored systems by either:
 
-## Install PMM Server
+    - using separate Kubernetes clusters for monitoring and databases
+    - configuring workload separation through node configurations, affinity rules, and label selectors
 
-??? info "Summary"
+2. Enable [high availability](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/ha-topology/) to ensure continuous monitoring during node failures
 
-    !!! summary alert alert-info ""
-        - Setup pmm-admin password
-        - Install
-        - Configuration parameters
-        - PMM environment variables
-        - PMM SSL certificates
-        - Backup
-        - Upgrade
-        - Restore
-        - Uninstall
+## Installation PMM Server on your Kubernetes cluster
 
-    ---
+Create the required Kubernetes secret and deploy PMM Server using Helm:
+{.power-number}
 
-### Set up pmm-admin password
+1. Create Kubernetes secret to set up `pmm-admin` password:
+    ```sh
+    cat <<EOF | kubectl create -f -
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: pmm-secret
+      labels:
+        app.kubernetes.io/name: pmm
+    type: Opaque
+    data:
+    # base64 encoded password
+    # encode some password: `echo -n "admin" | base64`
+      PMM_ADMIN_PASSWORD: YWRtaW4=
+    EOF
+    ```
 
-Create Kubernetes secret with pmm-admin password:
-```sh
-cat <<EOF | kubectl create -f -
-apiVersion: v1
-kind: Secret
-metadata:
-  name: pmm-secret
-  labels:
-    app.kubernetes.io/name: pmm
-type: Opaque
-data:
-# base64 encoded password
-# encode some password: `echo -n "admin" | base64`
-  PMM_ADMIN_PASSWORD: YWRtaW4=
-EOF
-```
+2. Get admin password:
 
-To get admin password execute:
+    ```sh
+    kubectl get secret pmm-secret -o jsonpath='{.data.PMM_ADMIN_PASSWORD}' | base64 --decode
+    ```
 
-```sh
-kubectl get secret pmm-secret -o jsonpath='{.data.PMM_ADMIN_PASSWORD}' | base64 --decode
-```
+3. Add the Percona repository and deploy PMM Server with default settings and your secret. See configuration parameters for customization. See [configuration parameters]((#view-available-parameters)) for customization.
 
-### Install
+    ```sh
+    helm repo add percona https://percona.github.io/percona-helm-charts/
+    helm install pmm \
+    --set secret.create=false \
+    --set secret.name=pmm-secret \
```

Comment:
```suggestion
    helm install pmm \
    --set secret.create=false \
    --set secret.name=pmm-secret \
    --version ^1.4.0 \
```
this will make sure that user installs 1.4.0+ version.
I think we should mention helm chart version in this block, not in pre-requisites.